virtual_rainforest/core/base_model.py

```diff
@@ -279,6 +382,8 @@ def __init_subclass__(cls) -> None:
         try:
             cls._check_model_name()
             cls._check_required_init_vars()
+            cls._check_time_bounds_units()
+            cls._check_time_bounds_values()
```

Comment:
And then here we could do:

```python
lwbnd = cls._check_time_bounds_units('lower_bound_on_time_scale')
upbnd = cls._check_time_bounds_units('upper_bound_on_time_scale')
```

And then outside of the try just run the separate check for `lwbnd <= upbnd`.

Reply:
That probably makes `_check_time_bounds_values()` small enough to simply move the actual check back into `__init_subclass__`?